test/curve.test.js

```diff
@@ -51,48 +46,59 @@ describe("check curve initialisation", () => {
         function run() {
             return initECSettings(activeCurveParams);
         }
+
```

Comment:
Nit: can we remove these empty lines before and after `run()` to be consistent with the other files?